src/app/course-page/course-search/course-search.component.spec.ts

```diff
@@ -19,7 +20,14 @@ describe('CourseSearchComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
+  it('should create CourseSearchComponent', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should search course-card', () => {
+		spyOn(component.search, 'emit');
+		const searchButton = fixture.debugElement.query(By.css('.search-button'));
+		searchButton.triggerEventHandler('click', null);
+		expect(component.search.emit).toHaveBeenCalledWith(undefined);
```

Comment:
why `undefined`? Test for a specific value to be emited

Reply:
fixed